src/TelemetryChannels/ServerTelemetryChannel/Shared/Shared.projitems

```diff
@@ -17,6 +17,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\ApplicationFolderProvider.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\ApplicationLifecycleTransmissionPolicy.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\ApplicationStoppingEventArgs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Implementation\BreezeResponse.cs" />
```

Comment:
Should we not reference Breeze which is internal codename in public sources? Perhaps call it something neutral, e.g. DataCollector?